docs/src/1_Mechanics/03_CharacterCreation.md

```diff
@@ -132,10 +145,12 @@ character creation, a player may only add newly awarded Experience points.
 | 9     | 5        |
 | 10    | 6        |
 
-The lower limit for Modifiers is -2. If -1, any Check made with that modifier must be 
-be a Color Match for success. If -2, any Check made with that modifier must be Suit 
+The lower limit for Modifiers is -2. If a Modifier is -1, any Check made with that modifier must be 
+be a Color Match for success. If -2, any Check made with that modifier must be a Suited 
 Match for success.
 
+When modifying an Attribute, the related Skills are affected as well. This means that boosting Strength to a +1 will also add a +1 to Brute and Intimidation. Reducing an Attribute by -1 also reduces the related Skills by -1. 
```

Comment:
Setting my attrib to -2 at level 1 would mean an extra 4 xp do use elsewhere, right?

Reply:
Yea I think -1 would yield +2 XP and -2 would yield +4XP, since I would consider -1 a minor vulny and -2 a major Vulny. 

It does beg the question of whether or not we create an attribute Vulny that is what actually allows you to reduce your attribute below 0 so it all feeds into that same +4XP cap at character creation or if we let someone take vulnys (+4) and then reduce two attributes to -2 (+8) and start with an additional +12 total XP versus someone who just takes vulny's being nearly 3 levels behind in XP than the person who dumps two stats for 4XP each.